groups/groups.yaml

```diff
@@ -1026,7 +1051,7 @@ groups:
       WhoCanApproveMembers: "ALL_MANAGERS_CAN_APPROVE"
       WhoCanModifyMembers: "OWNERS_AND_MANAGERS"
       WhoCanModerateMembers: "OWNERS_AND_MANAGERS"
-      ReconcileMembers: "false"
+      ReconcileMembers: "true"
```

Comment:
This will remove 30 members from the list. Is this intended?

Reply:
Dropped this commit.